source/guide_teamspeak.rst

```diff
@@ -0,0 +1,286 @@
+
+The target audience for TeamSpeak is gamers, who can use the software to communicate with other players on the same team of a multiplayer video game. Communicating by voice gives a competitive advantage by enabling players to keep their hands on the controls.
+
+Teamspeak is an alternativ to Mumble_ and needs a license to run. This can be optained from the the Teamspeak-License_ website.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Virtual Ports <basics-ports>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://www.teamspeak.com/en/features/licensing/
+
+
+Prerequisites
+=============
+
+.. include:: includes/open-port.rst
+
+Run the commmand four times to retrieve four ports. One open port for the ``default_voice_port``, ``filetransfer_port``, ``query_port`` and ``query_ssh_port``.
+
+
+Installation
+============
+
+Step 1
+------
+
+Download the Teamspeak Server for Linux. Get the latest download-URL from the Teamspeak-Downloads_ website.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir ts3server
+  [isabell@stardust ~]$ cd ts3server
+  [isabell@stardust ts3server]$ wget https://files.teamspeak-services.com/releases/server/3.11.0/teamspeak3-server_linux_amd64-3.11.0.tar.bz2
+  --2020-02-14 21:23:12--  https://files.teamspeak-services.com/releases/server/3.11.0/teamspeak3-server_linux_amd64-3.11.0.tar.bz2
+  Auflösen des Hostnamen »files.teamspeak-services.com (files.teamspeak-services.com)«... 151.139.128.10
+  Verbindungsaufbau zu files.teamspeak-services.com (files.teamspeak-services.com)|151.139.128.10|:443... verbunden.
+  HTTP-Anforderung gesendet, warte auf Antwort... 200 OK
+  Länge: 9245357 (8,8M) [application/x-tar]
+  In »»teamspeak3-server_linux_amd64-3.11.0.tar.bz2«« speichern.
+  
+  100%[======================================================================================================>] 9.245.357   32,4MB/s   in 0,3s   
+  
+  2020-02-14 21:23:12 (32,4 MB/s) - »»teamspeak3-server_linux_amd64-3.11.0.tar.bz2«« gespeichert [9245357/9245357]
+  
+  [isabell@stardust ts3server]$ 
+
+Step 2
+------
+
+Unpack the downloaded archive into the current directory
+
+.. code-block:: console
+
+  [isabell@stardust ts3server]$ tar --extract --bzip2 --strip-components=1 --verbose --file teamspeak3-server_linux_amd64-3.11.0.tar.bz2
+  teamspeak3-server_linux_amd64/LICENSE
+  [...]
+  teamspeak3-server_linux_amd64/CHANGELOG
+  [isabell@stardust ts3server]$
+
+
+Configuration
+=============
+
+Step 1: Setup license agreement
+-------------------------------
+
+Before starting the server for the first time, you need to create a file named ``.ts3server_license_accepted`` in the working directory. It is enough to create the file. No need to add content. 
```

Comment:
Please also explicitly link the license people are accepting here and note that they should actually read it (even though nobody will, because I am sure it's 20 pages long).

Reply:
done. finally found the correct license website on the TeamSpeak website. updated text and put in a link to the license models.

Reply:
nice! thank you!